pkg/controller/migplan/gvk.go

```diff
@@ -84,19 +88,6 @@ func reportGVK(plan *migapi.MigPlan, incompatibleMapping map[string][]schema.Gro
 		}
 		incompatibleNamespaces = append(incompatibleNamespaces, incompatibleNamespace)
```

Comment:
This will create a new slice and assign to `incompatibleNamespaces`.  The Incompatible.Namespaces will never be populated.